### PR TITLE
User story/1507 as a user, i want to have way smaller file sizes on my drive

### PR DESCRIPTION
**Pull request by briemla**
OP#1507

### CODE REVIEW

**Comment by briemla** [comment]
See comment in OP about the usage of bzip2.

File: OTVision/transform/transform.py [hunk 1]
```diff
@@ -163,7 +163,7 @@ def read_tracks(tracks_file: Path) -> tuple[pd.DataFrame, dict]:
     """
 
     # Read dicts and turn tracks into DataFrame
-    tracks_dict = read_json(tracks_file, filetype=tracks_file.suffix)
+    tracks_dict = read_json(tracks_file, filetype=tracks_file.suffix, decompress=False)
```
Comment (martinbaerwolff) [inline OTVision/transform/transform.py:166]: Wouldn't it be nice to automatically set `decompress` according to the input files? (file extension)

File: OTVision/helpers/files.py [hunk 2]
```diff
@@ -146,13 +151,20 @@ def _remove_dir(dir_to_remove: Path) -> None:
     dir_to_remove.rmdir()
 
 
-def read_json(json_file: Path, filetype: str = ".json") -> dict:
+def read_json(
+    json_file: Path,
+    filetype: str = ".json",
+    decompress: bool = True,
```
Comment (martinbaerwolff) [inline OTVision/helpers/files.py:157]: Should we add this to the config?

**Review by martinbaerwolff — COMMENTED** [review]
I had to make some bug fixes, as CI/CD tests failed. Also I have two questions in the code :)

File: OTVision/helpers/files.py (hunk 2 above)
Comment (briemla) [reply OTVision/helpers/files.py:157]: As it is a helper method, I would not add it to the config. After we defined the output format. We can decide which output formats should be compressed and add one config option for each file format.

File: OTVision/transform/transform.py (hunk 1 above)
Comment (briemla) [reply OTVision/transform/transform.py:166]: See comment on next discussion.

**Review by martinbaerwolff — APPROVED** [review]

**Comment by martinbaerwolff** [comment]
Ready to merge from my side!

**Pull request merged**